client/package.json

```diff
@@ -8,7 +8,8 @@
     "react-dom": "^16.8.4",
     "react-router-dom": "^5.0.0",
     "react-scripts": "2.1.8",
-    "styled-components": "^4.1.3"
+    "styled-components": "^4.1.3",
+    "@babel/runtime": ""
```

Comment:
What is this doing?

Reply:
Same question as @jema28

Reply:
I got error for @babel/runtime so I had to add this to avoid the error